interactive-clap-derive/src/derives/interactive_clap/structs/clap_for_named_arg_enum.rs

```diff
@@ -0,0 +1,62 @@
+use proc_macro2::Span;
+use proc_macro_error::abort_call_site;
+use quote::{quote, ToTokens};
+use syn;
+
+/// returns the whole result `TokenStream` of derive logic of containing module
+pub fn token_stream(ast: &syn::DeriveInput, fields: &syn::Fields) -> proc_macro2::TokenStream {
+    let name = &ast.ident;
+    fields
+        .iter()
+        .find_map(|field| field_transform(name, field))
+        .unwrap_or(quote!())
```

Comment:
I know it was before, but should we put here that debug macro? to indicate that we ccouldn't transform?

Reply:
i don't quite understand the suggestion. 
if the `dbg_cond!` macro is implied, i only added it to the paths i had trouble with figuring out for adding tests/looking at a bug or just generally figuring them out.
there many more possible paths to cover with debug statements, i think there're at least 100x more of them.

Reply:
i beleive this module was a product of extracting this chunk of code 
https://github.com/near-cli-rs/interactive-clap/pull/26/files#diff-f36bdc4e97c2bde45b6fdb5764c7a15faf2ab71c0fcd30d123dac570724594c0L147-L192 . 
(Github UI doesn't seem to be capable of autoexpanding this link to the exact chunk, you would have to scroll to `interactive-clap-derive/src/derives/interactive_clap/mod.rs` file and click `Load file` button to be able to see the exact chunk, without clicking any other links or buttons, or Github will overwrite the info about chunk in URL).